package protoFile.guild.TransferCorpsOrKickMember;
message TransferCorpsOrKickMemberRequest{
     required int32 id=1;//角色id
     required int32 operType = 2;//操作类型0移交军团长1踢出成员
     required int32 memberId=3;//成员id
}
message TransferCorpsOrKickMemberResponse{
     required bool result = 1;
     optional string message = 2;
}